web/src/main/java/com/futurewei/alcor/web/entity/gateway/GatewayIpJson.java

```diff
@@ -0,0 +1,28 @@
+package com.futurewei.alcor.web.entity.gateway;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class GatewayIpJson {
+
```

Comment:
Should we include "zeta" in the class name? since it's dedicated for zeta gateway only.